app/core/transport/http_client.py

```
@@ -71,3 +72,43 @@ def create_aiohttp_session(**session_kwargs) -> aiohttp.ClientSession:
         logger.debug("Created aiohttp session without proxy")
 
     return aiohttp.ClientSession(**session_kwargs)
+
+
+def setup_daily_proxy_environment():
+    """Set up proxy environment variables for Daily native client"""
+    import os
```

Ruff (0.13.1)

Redefinition of unused `os` from line 5

Remove definition: `os`

(F811)